### ISSUE TITLE
Report bugs to Nanelo DNS API

### ISSUE THREAD

**Issue by AnTheMaker** [acmesh-official/acme.sh#4519]
Please report bugs you come across when using the [Nanelo](https://nanelo.com) DNS Integration here.

**Comment by AnTheMaker** [comment]
Good catch, @QuadPiece - thank you! I've fixed it and created a PR which adds support for team API keys!

**Comment by neilpang** [comment]
Closing this tracking issue for now to keep the issue list clean. It remains the place to report problems with this provider -- if you hit a bug, comment here and the issue will be reopened.
